xml/System/AppDomain.xml

```diff
@@ -9046,7 +9045,11 @@
         <exception cref="T:System.ArgumentNullException">
           <paramref name="domain" /> is <see langword="null" />.</exception>
         <exception cref="T:System.CannotUnloadAppDomainException">
-          <paramref name="domain" /> could not be unloaded.</exception>
+.NET Core and .NET 5+ only: In all cases.
```

Comment:
What about adding a link to a bit of info as to why it's thrown for .NET Core/5? 

```suggestion
.NET Core and .NET 5+ only: In all cases. For more information, see [Application Domains](/dotnet/core/porting/net-framework-tech-unavailable?toc=/dotnet/fundamentals/toc.json&bc=/dotnet/breadcrumb/toc.json#application-domains)
```

Reply:
Good idea. I added it as a see also link instead of putting it in with the exception (as I haven't seen links in exception text anywhere else).